lnwallet/channel.go

```diff
@@ -859,6 +882,12 @@ func (lc *LightningChannel) diskHtlcToPayDesc(feeRate chainfee.SatPerKWeight,
 		theirWitnessScript: theirWitnessScript,
 	}
 
+	htlc.BlindingPoint.WhenSome(func(b tlv.RecordT[
```

Comment:
`WhenSomeV` is useful here again as it gives you just `*btcec.PublicKey` in this case.